tests/llm/test_metadata.py

```
@@ -122,12 +124,12 @@ class RopeScalingScenario:
     ModelInitScenario(
         id="llama32",
         model_class=Llama32,
-        model_path="meta-llama/Llama32-1B",
+        model_path="meta-llama/Llama-3.2-1B-Instruct",
         expected_template="user\n {instruction} {schema} \n assistant\n{prefill}",
         expected_add_bos=False,
         expected_add_eos=False,
-        expected_bos_token="<|im_start|>",
-        expected_bos_token_id=151644,
+        expected_bos_token="<s>",
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to argument: "expected_bos_token"

(S106)